data/remnant/remnant missions.txt

```diff
@@ -3946,40 +3946,35 @@ event "remnant: postverta reveal"
 	unvisit Caeculus
 	unvisit Prosa
 	unvisit Egeria
-	system Postverta
-		pos 297 269
-		link Diespiter
-		link Prosa
-		link Statina
-	system Diespiter
-		link Postverta
-		link Statina
-	system Statina
-		link Postverta
-		link Diespiter
-	system Levana
-		add object wormhole_barren
-			distance 2313
-			period 389
-			sprite planet/wormhole-red
 	system Caeculus
-		add object wormhole_barren2
+		add object "Wormhole Barren Beta"
+			sprite planet/wormhole-red
 			distance 2400
 			period 391
+	system Egeria
+		add object "Wormhole Barren Alpha"
 			sprite planet/wormhole-red
-	system Prosa
-		link Postverta
-		link Vaticanus
-		add object wormhole_barren2
 			distance 2313
 			period 389
+	system Levana
+		add object "Wormhole Barren Alpha"
 			sprite planet/wormhole-red
-	system Egeria
-		add object wormhole_barren
 			distance 2313
 			period 389
+	system Postverta
+		pos 297 269
+		link Diespiter
+		link Prosa
+		link Statina
+	system Prosa
+		add object "Wormhole Barren Beta"
 			sprite planet/wormhole-red
-		
+			distance 2313
+			period 389
+	link Postverta Diespiter
+	link Postverta Prosa
+	link Postverta Statina
```

Comment:
I still think you should have the systems linked from game start and have Postverta's starting position be a little more up and to the right so that it's out of jump range at game start. You would still move Postverta to `pos 297 269` with this event, though. 

I don't see unvisiting the systems as necessary, since they can't be visited in the first place before this event, but if you do what I suggested then it might make sense to unvisit Postverta as well.